### DISCUSSION TITLE
Dev mode won't work

### DISCUSSION THREAD

**Discussion by Ramortada** [opencardev/crankshaft#616] [General]
I managed to get into dev mode. I put in "pi" as username, but when it comes to putting the password, I can't type any letters. I can only press enter, and then it tells me invalid login. Why can't I type in the password?

**Comment by Novocaine85** [comment]
Use default username pi and password raspberry

https://github.com/opencardev/crankshaft/wiki/Crankshaft-dev-mode

it's a normal behaviour that you can't see any character when digiting the password.
Just type the password and press enter.

**Reply by Ramortada** [reply]
Thank you!
